Xamarin.Forms.Core/Device.cs

```diff
@@ -58,7 +58,7 @@ public static double GetNamedSize(NamedSize size, Type targetElementType)
 			return GetNamedSize(size, targetElementType, false);
 		}
 
-		public static void OnPlatform(Action iOS = null, Action Android = null, Action WinPhone = null, Action Default = null)
+		public static void OnPlatform(Action iOS = null, Action Android = null, Action WinPhone = null, Action Default = null, Action Tizen = null)
```

Comment:
Thanks for your comments. That's why I want to handle it on different PR. 
How do you feel about separating from this PR? Device.OnPlatform methods are also required to Tizen backend. However, at the moment, TargetDevice.Tizen is first step to support Xamarin.Forms.Platform.Tizen (Forms.cs) I think.